### PR TITLE
netlify: add page

### PR DESCRIPTION
**Pull request by scttrgd**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/netlify.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# netlify
+
+> Depoy sites and configure continuous deployment to the Netlify platform.
```
Comment (schneiderl) [inline pages/common/netlify.md]:
```suggestion
> Deploy sites and configure continuous deployment to the Netlify platform.
```

File: pages/common/netlify.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# netlify
+
+> Deploy sites and configure continuous deployment to the Netlify platform.
+> More information: <https://www.netlify.com/docs/cli/>.
+
+- Login to your Netlify account:
+
+`netlify login`
+
+- Create a new deploy from the contents of a folder:
```
Comment (agnivade) [inline pages/common/netlify.md]: We can just say "Deploy the contents of a directory:"

File: pages/common/netlify.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`netlify deploy`
+
+- Configure continuous deployment for a new or existing site:
```
Comment (agnivade) [inline pages/common/netlify.md]: or _an_ existing site

File: pages/common/netlify.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# netlify
+
+> Deploy sites and configure continuous deployment to the Netlify platform.
+> More information: <https://www.netlify.com/docs/cli/>.
+
+- Login to your Netlify account:
```
[the remark this answers is not in the document]
Comment (scttrgd) [reply pages/common/netlify.md]: So would it be _login to the Netlify account_?

File: pages/common/netlify.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# netlify
+
+> Deploy sites and configure continuous deployment to the Netlify platform.
+> More information: <https://www.netlify.com/docs/cli/>.
+
+- Login to a Netlify account:
+
+`netlify login`
+
+- Create a new deploy from the contents of a directory:
```
Comment (sbrl) [inline pages/common/netlify.md]: I'm not sure this sounds right. What about this?
```suggestion
- Create a new deployment from the contents of a directory:
```

...or this?
```suggestion
- Deploy the contents of a directory to Netlify:
```

File: pages/common/netlify.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (agnivade) [reply pages/common/netlify.md]: Correct.

File: pages/common/netlify.md (hunk 5 above)
Comment (agnivade) [reply pages/common/netlify.md]: That's what I said in my review comment too ;)

File: pages/common/netlify.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (scttrgd) [reply pages/common/netlify.md]: alrighty, I've added that change :)

[the remark this answers is not in the document]
Comment (scttrgd) [reply pages/common/netlify.md]: https://github.com/tldr-pages/tldr/pull/3374/commits/6f54487c699d2b891675f2a18cc903945d2ec25b

**Pull request merged**